src/main/java/mate/jdbc/model/Car.java

```diff
@@ -0,0 +1,69 @@
+package mate.jdbc.model;
+
+import java.util.Objects;
+
+public class Car {
+    private Long id;
+    private Long manufacturerId;
+    private String model;
```

Comment:
```suggestion
    private Long id;
    private String model;
    private Manufacturer manufacturer;
    private List<Driver> drivers;
```

car should contain Manufacturer and List of drivers